deployment/terraform/assets.go

```diff
@@ -39,6 +40,56 @@ func MustAssetString(name string) string {
 
 // RestoreAssets writes an embedded asset to the given directory
 func RestoreAssets(dir, name string) error {
+	assetPath := "assets/" + name
```

Comment:
Why not use `filepath.Join` here?
```suggestion
	assetPath := filepath.Join("assets", name)
```